web/app.fluidity.money/app/routes/$network/query/dashboard/home.tsx

```diff
@@ -31,29 +31,53 @@ export const loader: LoaderFunction = async ({ request, params }) => {
   const timestamp = new Date().getTime();
 
   const mainnetId = 0;
-  const infuraRpc = config.drivers[network][mainnetId].rpc.http;
 
-  const provider = new JsonRpcProvider(infuraRpc);
+  const prizePoolPromise: Promise<number> = (() => {
+    switch (chainType(network)) {
+      case "evm": {
+        const ethInfuraRpc = config.drivers["ethereum"][mainnetId].rpc.http;
 
-  const rewardPoolAddr = config.contract.prize_pool[network] ?? "";
+        const ethProvider = new JsonRpcProvider(ethInfuraRpc);
+
+        const ethRewardPoolAddr = config.contract.prize_pool["ethereum"] ?? "";
```

Comment:
might be better to map over supported network so it doesn't get messy if more networks are supported